xml/operations-maintenance-controller-swiftrings_recovery.xml

```diff
@@ -6,137 +6,123 @@
+   <step>
     <para>
-     Execute the restore job:
+     Create a temporary directory and extract the TAR archive (for example,
+     <filename>swring.tar.gz</filename>).
     </para>
-<screen>&prompt.root;freezer-agent --config ./restore_config.ini</screen>
+    <screen>&prompt.ardana;mkdir /tmp/swift_builder_dir_restore; sudo tar -z \
+--incremental --extract --ignore-zeros --warning=none --overwrite --directory \
+/tmp/swift_builder_dir_restore/  -f swring.tar.gz</screen>
     <para>
-     You now have the Swift rings in
+     You now have the &swift; rings in
      <literal>/tmp/swift_builder_dir_restore/</literal>
     </para>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
-     If the SWF-PRX[0] is already deployed, copy the contents of the restored
-     directory (<literal>/tmp/swift_builder_dir_restore/<replaceable>CLOUD_NAME</replaceable>/<replaceable>CONTROL_PLANE_NAME</replaceable>/builder_dir/</literal>) to
-     <literal>/etc/swiftlm/<replaceable>CLOUD_NAME</replaceable>/<replaceable>CONTROL_PLANE_NAME</replaceable>/builder_dir/</literal> on the SWF-PRX[0] Then from
-     the &clm; run:
+     If the SWF-PRX--first-member is already deployed, copy the contents of the
+     restored directory
+     (<literal>/tmp/swift_builder_dir_restore/<replaceable>CLOUD_NAME</replaceable>/<replaceable>CONTROL_PLANE_NAME</replaceable>/builder_dir/</literal>)
+     to
+     <literal>/etc/swiftlm/<replaceable>CLOUD_NAME</replaceable>/<replaceable>CONTROL_PLANE_NAME</replaceable>/builder_dir/</literal>
+     on the SWF-PRX--first-member
     </para>
+   </step>
+    <step>
+     <para>
+      Then from the &clm; run:
+     </para>
+    </step>
+    <step>
 <screen>&prompt.ardana;sudo cp -pr /tmp/swift_builder_dir_restore/<replaceable>CLOUD_NAME</replaceable>/<replaceable>CONTROL_PLANE_NAME</replaceable>/builder_dir/* \
-    /etc/swiftlm/<replaceable>CLOUD_NAME</replaceable>/<replaceable>CONTROL_PLANE_NAME</replaceable>/builder_dir/</screen>
+/etc/swiftlm/<replaceable>CLOUD_NAME</replaceable>/<replaceable>CONTROL_PLANE_NAME</replaceable>/builder_dir/</screen>
     <para>For example</para>
 <screen>&prompt.ardana;sudo cp -pr /tmp/swift_builder_dir_restore/entry-scale-kvm/control-plane-1/builder_dir/* \
-    /etc/swiftlm/entry-scale-kvm/control-plane-1/builder_dir/</screen>
+/etc/swiftlm/entry-scale-kvm/control-plane-1/builder_dir/</screen>
 <screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
 &prompt.ardana;ansible-playbook -i hosts/verb_hosts swift-reconfigure.yml</screen>
-   </listitem>
-   <listitem>
+   </step>
+   <step>
     <para>
      If the SWF-ACC[0] is<emphasis role="bold"> not </emphasis>deployed, from
      the &clm; run these playbooks:
     </para>
 <screen>&prompt.ardana;cd ~/scratch/ansible/next/ardana/ansible
 &prompt.ardana;ansible-playbook -i hosts/verb_hosts guard-deployment.yml
 &prompt.ardana;ansible-playbook -i hosts/verb_hosts osconfig-run.yml --limit &lt;SWF-ACC[0]-hostname&gt;</screen>
```

Comment:
Though here the SWF-ACC[0] may work better as we want to append '-hostname' to that value